src/components/Planner.vue

```diff
@@ -75,12 +82,23 @@ export default {
         document.querySelectorAll('tbody tr').forEach(tr => {
           if (!tr.children[4]) return
           const cod = tr.children[1].innerText.split('-')[0].trim()
-          if (cods.includes(cod)) {
+          if (!cods.includes(cod)) {
             tr.style.display = checked ? 'none' : ''
           }
         })
       }
     },
+    getDisciplinasNaoCursadas(event) {
+      const { checked } = event.target
+      const cods = this.cursadas.map(({ id }) => id)
+      document.querySelectorAll('tbody tr').forEach(tr => {
+        if (!tr.children[4]) return
```

Comment:
```suggestion
        const minhaCondicaoComUmNomeLegal = !tr.children[4]
        if (minhaCondicaoComUmNomeLegal) return
```

É sempre legal usar nomes pra representar essas condições que refletem uma lógica de negócio implícita

Reply:
Legal, super concordo! Nesse caso infelizmente eu não consegui simular esse cenário então não entendi bem o que ela checa :/ daí apenas copiei do código mais antigo e mantive;